back-end/sentiment-api/resources/client_private.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
These keys are used to mock the sts server right? Should we include them in the `tests/resources` directory?